code/game/objects/items/pneumaticCannon.dm

```diff
@@ -82,21 +90,19 @@
 	return TRUE
 
 /obj/item/pneumatic_cannon/wrench_act(mob/living/user, obj/item/tool)
+	if(needs_air == FALSE)
+		return
 	playsound(src, 'sound/items/ratchet.ogg', 50, TRUE)
-	switch(pressureSetting)
-		if(1)
-			pressureSetting = 2
-		if(2)
-			pressureSetting = 3
-		if(3)
-			pressureSetting = 1
-	to_chat(user, span_notice("You tweak \the [src]'s pressure output to [pressureSetting]."))
+	pressureSetting = pressureSetting >= HIGH_PRESSURE ? LOW_PRESSURE : pressureSetting + 1
+	balloon_alert(user, "output level set to [pressureSetting]")
```

Comment:
Should be giving an actual word here, like "low/medium/high"

Reply:
Should i change it for powerfist too then?

Reply:
Would be fine